marklogic-data-hub-central/ui/src/components/entities/mapping/source-entity-map/source-to-entity-map.tsx

```diff
@@ -642,7 +640,7 @@ const SourceToEntityMap = (props) => {
                         <Table
                             pagination={false}
                             className={styles.entityTable}
-                            scroll={{  x: 'max-content', y: '60vh' }}
+                            scroll={{ y: '60vh' }}
```

Comment:
Gave a comment about this above. Let's not remove the x axis scroll value unless its causing an issue.